src/charm.py

```diff
@@ -48,6 +53,27 @@ def __init__(self, *args):
         self.framework.observe(
             self.on.content_cache_pebble_ready, self._on_content_cache_pebble_ready
         )
+        self.framework.observe(
+            self.on.nginx_prometheus_exporter_pebble_ready,
+            self._on_nginx_prometheus_exporter_pebble_ready,
+        )
+        # Provide ability for Content-cache to be scraped by Prometheus using prometheus_scrape
+        self._metrics_endpoint = MetricsEndpointProvider(
+            self, jobs=[{"static_configs": [{"targets": ["*:9113", "*:9102"]}]}]
```

Comment:
I can get why port 9113 is a target, but why port 9102 ?

Reply:
if we need a statsd exporter sometime, that is the port used for it. I'll delete it for now anyway, thank you for noticing!